py-polars/polars/frame.py

```diff
@@ -305,7 +307,7 @@ def to_pandas(
 
     def to_csv(
         self,
-        file: Union[TextIO, str, Path],
+        file: Union[TextIO, str, Path] = None,
```

Comment:
Now that we also accept a `None`, this should be `Optional[Union[TextIO, str, Path]]`

Reply:
Ok, I get it now, thanks！